app/controllers/api/v1/parks_controller.rb

```diff
@@ -15,9 +15,24 @@ def create
             render json: { park: park, error: [], successStatus: "Submitted Successfully!"}
         else
             render json: { park: {}, error: park.errors.full_messages, successStatus: "" }
+        end
+    end
 
+    def edit
+    end
 
-            # @failure = park.errors.full_messages.to_sentence
-        end
+    def update
+
+      park = Park.find(params[:id])
+      if park.update(park_params)
+        render json: {park: park, error: ''}
+      else
+        render json: {park: park, error: ''}
```

Comment:
this does not provide the user with errors. If the error key is not going to be used, I recommend we just omit it. Also I recommend we include a `status: 422` here if validation fails